Tests/E2E/README.md

```diff
@@ -18,17 +18,54 @@ This guide helps you to execute and author E2E tests on your local environment.
  +---------+
 ```
 
-* LoRaWan solution up and running (IoT Edge Device, IoT Hub, LoRa Keys Azure Function, Redis, etc.)
+* LoRaWan solution up and running (IoT Edge Device, IoT Hub, LoRa Keys Azure Function, Redis, container registry etc.)
 * Seeeduino LoRaWan device (leaf test device) connected via USB to a computer where the LoRaWan.Tests.E2E will run.
 * Module LoRaWanNetworkSrvModule logging configured with following environment variables:
   * LOG_LEVEL: 1
   * LOG_TO_UDP: true
   * LOG_TO_UDP_ADDRESS: development machine IP address (ensure IoT Edge machine can ping it)
 * E2E test configuration (in file `appsettings.local.json`) has UDP logging enabled `"UdpLog": "true"`
 
-## Installation
+## Setup
 
-1. Connect and setup Seeduino Arduino with the serial pass sketch
+### LoRa Basics Station and LNS setup
+
+Configure LoRa Basics Station and Network Server to run on your local LoRa gateway:
+
+1. Copy `LoRaEngine/example.env` file and name it `.env`.
+
+    - Update Container Registry settings and facade function app section with details of your created registry and function app.
+    - Update LoRaWanNetworkSrvModule settings:
+        ```
+        NET_SRV_LOG_LEVEL=1
+        NET_SRV_LOGTO_HUB=true
+        NET_SRV_LOGTO_UDP=true
+        NET_SRV_LOG_TO_UDP_ADDRESS=<your-local-ip-address>
+        ```
+    - Update `LBS_TC_URI`:
+    
+        `LBS_TC_URI=ws://172.17.0.1:5000`
```

Comment:
@Mandur I'm not 100% if there was another place where this Uri needed to be configured, do you know?